local/assets/cloud-config/coreos.yml

```diff
@@ -34,13 +34,12 @@ coreos:
             # Setting max log size to 10M
             Environment="DOCKER_OPTS=--log-driver=json-file --log-opt max-size=10m"
 
-{{ if index .env "ssh_authorized_keys" }}
+{{ if index .host.metadata "ssh_authorized_keys" }}
```

Comment:
lets keep this one `.host.metadata.ssh_authorized_keys`. Then for each host in the inventory in this group give it the `key`